packages/common-common/src/schemas/subscription/subscriptionTypes.ts

```diff
@@ -0,0 +1,11 @@
+export enum NotificationCategories {
+  NewComment = 'new-comment-creation',
+  NewThread = 'new-thread-creation',
+  NewMention = 'new-mention',
+  NewReaction = 'new-reaction',
+  NewCollaboration = 'new-collaboration',
+  ThreadEdit = 'thread-edit',
+  CommentEdit = 'comment-edit',
+  ChainEvent = 'chain-event',
+  SnapshotProposal = 'snapshot-proposal',
+}
```

Comment:
this is duplicated with NotificationCategories defined in common-common/src/types. How do we want to go about it to avoid duplications like this? If we have common-common types, why do we need to defined it here again?

Reply:
more conventions